### PR TITLE
Makes some minor improvements or specifications in style guide

### PR DESCRIPTION
**Pull request by davidlmobley**
- Some OpenFF terms
- Numbering/naming conventions for figures
- Further clarify what should be in captions

### CODE REVIEW

**Comment by davidlmobley** [comment]
@Yoshanuikabundi want to review this one for me? The OpenFF naming bit was from discussions for the recent workshop.

File: Writing/StyleGuide.md [hunk 1]
```diff
@@ -26,6 +26,9 @@ But some issues are worth addressing specifically within the group because they
 - "Van der Waals": Describes the physical interaction/behavior, in contrast to Lennard-Jones, a particular potential function which can be used to describe this interaction.
 - "Pose": Usually used to discuss a static placement of a ligand/small molecule in a receptor. Refers primarily to the placement of the small molecule, NOT to the structure of the receptor.
 - "Binding mode": Refers to the overall placement/orientation/conformation of a small molecule bound to a receptor, and gives a less static picture than the word "pose". With care, the idea of "binding mode" can be generalized to include the relevant receptor conformation(s), bound waters, etc. It can also include a group of relevant microstates that correspond to a metastable state.
+- OpenFF-specific terms:
+  - BespokeFit is the name of the package for fitting custom torsions
+  - TorsionDrive scans torsions
```
Comment (Yoshanuikabundi) [inline Writing/StyleGuide.md:31]:
```suggestion
- OpenFF-specific terms:
  - Capitalize the first letter of each word in a compound word software name released by OpenFF:
      - "BespokeFit" is the name of the package for fitting custom torsions
      - "TorsionDrive" scans torsions
```

**Review by Yoshanuikabundi — APPROVED** [review]
Looks good! We had similar discussions around the release of BespokeFit. I have added one optional suggestion that explicitly describes how compound names of OpenFF software should be capitalized. I apparently can't get a hold of the ACS style guide but from googling it looks like it does recommend title case for section headers, so I guess I can live with that.

**Comment by davidlmobley** [comment]
Thanks!

**Pull request merged**